content/en/docs/setup/install/helm/index.md

```diff
@@ -130,7 +130,18 @@ $ helm template install/kubernetes/helm/istio --name istio --namespace istio-sys
 
 {{< /tab >}}
 
-{{< tab name="sds" cookie-value="sds" >}}
+{{< tab name="mTLS enabled" cookie-value="mtls" >}}
```

Comment:
```suggestion
{{< tab name="demo mTLS" cookie-value="mtls" >}}
```

Reply:
Frank, let's not call it "demo" since we have decided to not have two demos :)

Reply:
The instruction says to choose a configuration profile (https://deploy-preview-5482--preliminary-istio.netlify.com/docs/setup/additional-setup/config-profiles/), so you can't just say "Mutual TLS" on the label, since there's no such profile.

Currently your tab is installing the `demo` profile with mTLS enabled. If you don't want to label it as such, then I suggest changing the contents of that tab to say that you can turn on mTLS for any configuration profile and explain what options the user needs to set and then give the instruction as just an example. Take a look at the CNI tab, it is written that way.

I would also change the SDS tab back to just `sds`, like it was before, because the configuration profiles already says that the sds profile turns on mutual TLS.

I would also suggest putting the Mutual TLS tab second last, after `sds` but before `CNI`, so the first 4 are straight profile selections, while the last 2 are how to turn on a special feature with any profile.